### DISCUSSION TITLE
Show current state about accessories

### DISCUSSION THREAD

**Discussion by brutella** [brutella/hkknx-public#299] [Ideas]
I'm working on improving the web ui to show the current state of accessories. Currently you can only create and edit accessories in the web ui.

Starting with version [2.9.0-b1](https://github.com/brutella/hkknx-public/releases/tag/2.9.0-b1), the web ui shows live information of accessories – e.g. the current temperature of thermostat, the state of lightbulbs and the current position of blinds. Some accessories will be interactive, meaning that lightbulbs can be toggled, and blinds can be opened/closed and moved to a specific position. All of this can be done in the *Accessories* tab. 

This will enable desktop users to control accessories without using HomeKit. After editing an accessory, the configuration can be verified by controlling the accessories right in the web ui.

The web ui should also be useable from any mobile devices. This will enable Android users to control accessories without having to go through HomeKit – which is not available on Android.

Looking for feedback to make this a great new feature.

**Comment by mbrockeu** [comment]
Mein erstes Feedback, es funktioniert super und nun kann man auch mal von einem anderen Gerät die Lampen an und ausschalten.
Da kommen wir dem Thema Dashboard ja schon wieder näher.

Was mir persönlich fehlt ist ein Filter um deaktivierte Geräte rauszunehmen. Ich habe zwei Instanzen von HKKNX und in der ersten im Prinzip alle Geräte drin einen Großteil aber deaktiviert da ich über 200 Devices/Sensoren drin habe. Die stören in der Übersicht ein wenig.

**Reply by brutella** [reply]
Danke für das Feedback.

Was für Filter (neben dem Ausblenden von inaktiven Geräten) würdest du dir wünschen? So ähnlich wie die Kategorien in der Apple Home-App – also Lampen, Klima (Temperatur, Feuchtigkeit), Sicherheit (Schlösser, Alarmanlage, Bewegungssensoren) – oder etwas anderes?

**Reply by mbrockeu** [reply]
Die Filter hast du ja im Prinzip schon da, zumindest für die Geräteansicht. Was eher noch sinnvoll wäre eine Kombination zum Beispiel, also Raum + Gerätetyp(en).

Wenn man dann die Ansicht noch in ein Kachelmodus umstellen kann zum Steuern hätte man schon ein schönes Dashboard für einen Raum. Das ganzen mit einem Deeplink gekoppelt wäre ja schnell auf ein Tablet gebracht so dass z.B. Gäste/Kinder nur Geräte in Raum XY steuern können.

Sofern das hier in die Thematik passt.

**Reply by brutella** [reply]
Ja, so etwas ist auf jeden Fall angedacht. Somit könnte man über das Dashboard nur gewissen Geräte steuern – was ja mit HomeKit bis jetzt nicht umsetzbar ist.

**Reply by mbrockeu** [reply]
Mir fällt in dem Zusammenhang gerade ein, dass es natürlich schön wäre mein Ticket #253  mal mit reindenken kann. 

Es wäre halt am Ende deutlich einfacher wenn man eine Instanz zum verwalten hat mit allen Geräten und diese ggfs. Virtuellen bridges zuordnen kann. Gerade wenn man dann ein „Dashboard“ hat sind wenigstens alle Geräte drin. Ich habe bei mir aktuell schaltbare Geräte und Sensoren getrennt.
